debian/control

```diff
@@ -35,7 +35,6 @@ Depends: ${shlibs:Depends},
          ayatana-indicator-common,
          upower,
 Recommends: mate-indicator-applet | xfce4-indicator-plugin,
-            zenity,
```

Comment:
@tari01 So, let's discuss the design of this.

We should pull-in zenity somewhere. The best place for this would be in the renderers' Depends: fields.

However, do we want zenity to jump-in if all other external tools fail to start? What about Lomiri? Hard-wiring zenity there via indicators ( or libayatana-common )  is probably not a good idea.

Saying this, I haven't see an D: on zenity in libayatana-common either, so what the architecture idea of the zenity-consolidation code changes you are currently preparing for upcoming bundle release 23.10.0?

Reply:
> @tari01 So, let's discuss the design of this.
> 
> We should pull-in zenity somewhere. The best place for this would be in the renderers' Depends: fields.
> 
> However, do we want zenity to jump-in if all other external tools fail to start? What about Lomiri? Hard-wiring zenity there via indicators ( or libayatana-common ) is probably not a good idea.
> 
> Saying this, I haven't see an D: on zenity in libayatana-common either, so what the architecture idea of the zenity-consolidation code changes you are currently preparing for upcoming bundle release 23.10.0?

The indicators should definitely not depend on zenity because they are not using it, so drop it from all of them. Since the actual zenity calls reside in `libayatana-common`, it should have a zenity dependency. To avoid pulling it into Lomiri, we could use some `zenity | lomiri-thing` dependency magic.